podman-update/build/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM golang:1.16-stretch
 
 
 ARG DEBIAN_FRONTEND=noninteractive
-RUN apt-get update && apt-get install -y zip git pkg-config libc6-arm64-cross libc6-dev-arm64-cross binutils-aarch64-linux-gnu libncurses5-dev build-essential bison flex libssl-dev bc gcc-aarch64-linux-gnu
+RUN apt-get update && apt-get install -y libsystemd-dev zip git pkg-config libc6-arm64-cross libc6-dev-arm64-cross binutils-aarch64-linux-gnu libncurses5-dev build-essential bison flex libssl-dev bc gcc-aarch64-linux-gnu
```

Comment:
Duplicate to new DOckerfile called Dockerfile.Se

Reply:
gotcha. so the idea would be to create a separate podman binary.